cloudinit/config/schema.py

```diff
@@ -1429,15 +1499,30 @@ def handle_schema_args(name, args):
 
     error_types = []
     for idx, (cfg_type, cfg_file) in enumerate(config_files, 1):
+        performed_schema_validation = False
         if multi_config_output:
             print(f"\n{idx}. {cfg_type} at {cfg_file}:")
+        if cfg_type == "network-config":
+            cfg_schema = get_schema(cfg_type)
+            schema_type = cfg_type
+        else:
+            cfg_schema = full_schema
```

Comment:
It can be reconstructed here, but I was trying to reuse `full_schema` variable for user-data, vendor-data and vendor-data2 as that schema is the same for all 3 config file "types"